cmd/maestro/server/api_server.go

```diff
@@ -126,7 +126,7 @@ func NewAPIServer(eventBroadcaster *event.EventBroadcaster) Server {
 
 	// TODO: support authn and authz for gRPC
```

Comment:
remove this comment since we support it.

Reply:
removed this.